Ladybird/CMakeLists.txt

```diff
@@ -140,6 +144,13 @@ elseif (APPLE)
         -fobjc-arc
         -Wno-deprecated-anon-enum-enum-conversion # Required for CGImageCreate
     )
+elseif(ANDROID)
+    add_library(ladybird SHARED
+        ${SOURCES}
+        src/main/cpp/LadybirdActivity.cpp
```

Comment:
Flyby comment - Ladybird/src seems a bit high level for android-only sources, maybe move these to Ladybird/Android/src?

Reply:
yeah, I'm not quite sure how to get the `build.gradle.kt` to accept that path though 😅 . Any pointers there would be much appreciated

Reply:
Figured it out :)